src/sections/TokenSection.tsx

```diff
@@ -117,3 +147,44 @@ function CurrencyIcon({ currencyKey }: { currencyKey: CurrencyKey }) {
 
   return <Icon width={40} height={40} />;
 }
+
+const SkeletonTokenPlaceholder = () => {
```

Comment:
please move the code before the main component.

Reply:
done